tests/test_serializable.py

```diff
@@ -32,3 +60,138 @@ def test_setup_serialization(self) -> None:
     def test_setup_deserialization(self) -> None:
         # Act
         set_deserializers()
+
+    @settings(
+        deadline=timedelta(milliseconds=10000),
+        suppress_health_check=[HealthCheck.too_slow],
+        max_examples=5,
+        # disabling the "shrink" phase, because it runs very slowly
+        phases=[Phase.explicit, Phase.reuse, Phase.generate, Phase.target],
+    )
+    @given(
+        integers(1, 3).flatmap(lambda n: elections_and_ballots(n)), elements_mod_q(),
+    )
+    def test_eg_serialization(
```

Comment:
My goal was to write a stripped-down test that only exercises serialization/deserialization. It would be good to have a larger end-to-end test that also verifies the cryptographic tallies and proofs, and such a future test would subsume this one.